### ISSUE TITLE
Should suggest indenting continuation lines

### ISSUE THREAD

**Issue by l0b0** [koalaman/shellcheck#2589]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/usr/bin/env bash
foo bar baz
a bunch of arguments broken into \
multiple \
lines
ban bay bal
```

#### Here's what shellcheck currently says:

> No issues detected!

#### Here's what I wanted or expected to see:

Something like this:

> Indent continuation lines by 4 extra spaces for clarity.

[Rationale](https://paperless.blog/bash-lints-beyond-shellcheck): When continuation lines are indented the same as non-continuation lines it can be taxing to tell which lines belong with the previous one.

**Comment by koalaman** [comment]
I definitely agree, but `shfmt` already does this better. ShellCheck does not keep information about whitespace and indenting while building its AST, so it would additionally be hard to compete with an already excellent formatter built for these kinds of issues.
